pkg/codegen/internal/test/testdata/external-resource-schema/docs/cat.md

```diff
@@ -0,0 +1,449 @@
+
+---
+title: "Cat"
+title_tag: "example.Cat"
+meta_desc: "Documentation for the example.Cat resource with examples, input properties, output properties, lookup functions, and supporting types."
+---
+
+
+
+<!-- WARNING: this file was generated by test. -->
+<!-- Do not edit by hand unless you're certain you know what you are doing! -->
+
+
+
+
+## Create a Cat Resource {#create}
+{{< chooser language "typescript,python,go,csharp" / >}}
+
+
+{{% choosable language nodejs %}}
+<div class="highlight"><pre class="chroma"><code class="language-typescript" data-lang="typescript"><span class="k">new </span><span class="nx">Cat</span><span class="p">(</span><span class="nx">name</span><span class="p">:</span> <span class="nx">string</span><span class="p">,</span> <span class="nx">args</span><span class="p">?:</span> <span class="nx"><a href="#inputs">CatArgs</a></span><span class="p">,</span> <span class="nx">opts</span><span class="p">?:</span> <span class="nx"><a href="/docs/reference/pkg/nodejs/pulumi/pulumi/#CustomResourceOptions">CustomResourceOptions</a></span><span class="p">);</span></code></pre></div>
+{{% /choosable %}}
+
+{{% choosable language python %}}
+<div class="highlight"><pre class="chroma"><code class="language-python" data-lang="python"><span class=nd>@overload</span>
+<span class="k">def </span><span class="nx">Cat</span><span class="p">(</span><span class="nx">resource_name</span><span class="p">:</span> <span class="nx">str</span><span class="p">,</span>
+        <span class="nx">opts</span><span class="p">:</span> <span class="nx"><a href="/docs/reference/pkg/python/pulumi/#pulumi.ResourceOptions">Optional[ResourceOptions]</a></span> = None<span class="p">,</span>
+        <span class="nx">age</span><span class="p">:</span> <span class="nx">Optional[int]</span> = None<span class="p">,</span>
```

Comment:
The types in the signature here should be wrapped as `Optional[pulumi.Input[int]]`.

In the table of properties below, they should not be wrapped (this looks good now).

Reply:
> The types in the signature here should be wrapped as Optional[pulumi.Input[int]].

Were they wrapped before?